apps/medical-illustration/src/app/app-web-component.ts

```diff
@@ -0,0 +1,103 @@
+import { HttpClient } from '@angular/common/http';
+import { ChangeDetectionStrategy, Component, EventEmitter, inject, Input, OnChanges, Output } from '@angular/core';
+import { NodeMapEntry } from '@hra-ui/components/molecules';
+import { BehaviorSubject, Observable, tap } from 'rxjs';
+
+import { CellEntry, IllustrationData, JsonLd } from './models';
+
+/**
+ * Medical illustration web component
+ */
+@Component({
+  selector: 'hra-root-wc',
+  templateUrl: 'app-web-component.html',
+  styleUrls: ['app-web-component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AppWebComponent implements OnChanges {
+  /** Http client */
+  private readonly http = inject(HttpClient);
+
+  /** Ftu illustrations file: can be url or file object */
+  @Input() lookupSrc: string | JsonLd = '';
+
+  /** Illustration info: can be illustration id or illustration data object */
+  @Input() illustrationSrc: string | IllustrationData = '';
```

Comment:
Once fully published, we will set a default illustration source with a stable URL. Will that work here?

Reply:
By illustration source you mean the entire ftu illustrations file right? In that case you set the url as lookupSrc and it will fetch the data.

Reply:
Yes, that's right. When we get a stable URL for the jsonld file we will put that there.